modules/ROOT/pages/architecture.adoc

```diff
@@ -85,6 +85,11 @@ image::architecture-network.png[]
 
 This diagram shows the TCP load balancer used to load balance requests across the internal load balancers running on the controllers. It also shows the internal load balancer that distributes requests to each replica of Mule applications running on the workers.
 
+For internal load balancers:
+
+* *Shared Mode* is the default setting if no dedicated load balancer node is added in Runtime Fabric. In this mode, the internal load balancer runs on all controller nodes with the amount of CPU and memory specified.
+* *Dedicated Mode* specifies that the entire node is available, so you cannot specify the amount of cores and memory. If *Dedicated Mode* is disabled, then all nodes are already in sharing mode, which prevents them from being dedicated.
```

Comment:
Don't understand the meaning of the second sentence. If this is trying to explain that it is an A or B selector, it would be better to add something additional above like: If you have "Ingress Dedicated Nodes" (kubernetes labels the nodes as Ingress as does the UI under manage RTF), you can select either "Shared or Dedicated". If you do not have Ingress Dedicated Nodes, the "Dedicated Mode" option will be greyed out. Global to other similar text blocks below.

Reply:
Agree we can simplify this line. Consider the following:

*Dedicated Mode* is a setting that is available if one or more dedicated internal load balancer nodes are available in Runtime Fabric. All available resources in these nodes are used for running the internal load balancer, so the options to specify the amount of cores and memory are not available in this mode.

Reply:
Reworded this - thank you!